install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
```

Comment:
I'm not sure how my proposal goes against it.  I'm just saying that since `iptables` was the old default, that is still maintained as the default in current versions so that you don't end up with a mixed cluster when adding nodes...